sdk/purview/Azure.Analytics.Purview.Account/api/Azure.Analytics.Purview.Account.netstandard2.0.cs

```diff
@@ -0,0 +1,213 @@
+namespace Azure.Analytics.Purview.Catalog
```

Comment:
This doesn't seem like the API surface for the library.  Can you please run the `Export-Api.ps1` script to update this file (it will also fix a leg in CI)